libtenzir/builtins/operators/show.cpp

```diff
@@ -98,6 +98,18 @@ class plugin final : public virtual operator_plugin<show_operator> {
     parser.add(aspect, "<aspect>");
     parser.parse(p);
     if (aspect) {
+      // We moved `show version` to `version` when we made the `show` operator
+      // always report a remote location for consistency. This makes it less of
+      // a breaking change.
+      if (aspect->inner == "version") {
+        auto op = pipeline::internal_parse_as_operator("version");
+        if (not op) {
+          diagnostic::error("failed to parse `version` operator: {}",
+                            op.error())
+            .throw_();
+        }
```

Comment:
Shouldn't this be an assertion?

Reply:
You can run into this with this config file:

```yaml
tenzir:
  disable-plugins:
    - version
```